### ISSUE TITLE
Mesh that are delayed Join in the scene cannot trigger events

### ISSUE THREAD

**Issue by hexianWeb** [Tresjs/tres#758]
### Describe the bug

I wrote in the code
```js
onMounted(() => {
  setTimeout(() => {
    showRef.value = !showRef.value
  }, 2000)
})
```
&
```vue
  <!--mesh with click event-->
  <TresMesh :position="[2, 2, 0]" name="Father" @click="handleClick" v-if="showRef">
    <TresSphereGeometry :args="[1, 1]" />
    <TresMeshNormalMaterial />
    <Html :position="[0, 0, 0]" center>
      <span class="cool-font">Click delay event</span>
    </Html>
  </TresMesh>
```
I found out it couldn't trigger the event! 
This also happens with `primitive` load large `glb` file.
But interestingly, this error may be avoided by moving the mouse in `Canvas`after all `model or mesh` have been "loaded in the scene".

**The mouse is always in the canvas (Broken)**

https://github.com/Tresjs/tres/assets/59913119/e3171f55-ac9b-4f59-9848-3d5c714c42fe

**The mouse enter canvas after mesh "loaded"**

https://github.com/Tresjs/tres/assets/59913119/aa553c7a-355f-4d9d-a616-be3b99bf772e

### Reproduction

https://stackblitz.com/github/hexianWeb/starter/tree/event-register

### Steps to reproduce

If the user case is broken, please tell me in time

### Used Package Manager

pnpm

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/Tresjs/tres/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/Tresjs/tres/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://tresjs.org/guide).
- [X] Check that there isn't [already an issue](https://github.com/tresjs/tres/issues) that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by hexianWeb** [comment]
[useTresEventManager\index.ts](https://github.com/Tresjs/tres/blob/main/src/composables/useTresEventManager/index.ts#L127), objects added later will not appear in the `hits` collection

**Comment by alvarosabu** [comment]
Hey buddy @garrlker mind taking a look to this one?

**Comment by andretchen0** [comment]
## Possibly related

* #763

**Comment by garrlker** [comment]
@hexianWeb is this issue fixed for you in version 4.2.0?

**Comment by hexianWeb** [comment]
Yes, it has been fixed after version `tresjs 4.2.0`
